CHANGELOG.md

```diff
@@ -3,20 +3,41 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.32.0
+------------------------------
+*August 10, 2018*
+
+### Added
+- Peer dependency to `f-icons` added to `package.json`
+- CODEOWNERS file added
+
+### Changed
+- Chevron icon in language switcher now inlined using inlineSVG helper
+- Tidied up some class names in the language switcher
+- Updated PR template with new links and updated browsers
+
+### Fixed
+- The body container now doesn't jump around when the menu is opened.
+- The navigation height is correct when multi language versions of the site are viewed (it menu previously overlapped the header when open).
```

Comment:
Should this read?

(the menu previously overlapped the header when open)

Reply:
Yeh – didn't read correctly. Have updated